src/Utils/Strings.php

```diff
@@ -147,24 +152,26 @@ public static function toAscii(string $s): string
 
 		// transliteration (by Transliterator and iconv) is not optimal, replace some characters directly
 		$s = strtr($s, ["\u{201E}" => '"', "\u{201C}" => '"', "\u{201D}" => '"', "\u{201A}" => "'", "\u{2018}" => "'", "\u{2019}" => "'", "\u{B0}" => '^', "\u{42F}" => 'Ya', "\u{44F}" => 'ya', "\u{42E}" => 'Yu', "\u{44E}" => 'yu']); // „ “ ” ‚ ‘ ’ ° Я я Ю ю
-		if (ICONV_IMPL === 'glibc') {
+		if ($iconv !== 'libiconv') {
```

Comment:
What about opposite comparison?
```suggestion
		if ($iconv === null) {
```